cluster/etcd.go

```diff
@@ -21,6 +21,16 @@ func (c *Cluster) RestoreEtcdSnapshot(ctx context.Context, snapshotPath string)
 	if isEqual := c.etcdSnapshotChecksum(ctx, snapshotPath); !isEqual {
 		return fmt.Errorf("etcd snapshots are not consistent")
 	}
+
+	// get etcd snapshots from s3 if backup backend server is set
```

Comment:
This needs to _before_ the checksum check.

Reply:
resolved.